partials/logo.php

```diff
@@ -0,0 +1 @@
+<svg id="logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 820 48"><path d="M20.4 23.5L15.5 9.8H11v28.6h4.9V24.2l3.1 9.2h2.6l3.2-9.3v14.3h4.9V9.8h-4.5zm244 1.9c2.2-1.5 3.2-3.5 3.1-6v-2.6c0-1.9-.7-3.6-2-5-1.4-1.4-3-2-4.9-2h-7.4v28.6h4.9V26.5h1.6l3.7 11.8h5.2l-4.2-12.9zm-1.6-6c0 .8 0 .9-.1 1 0 .3-.2.6-.5.9-.5.5-1.1.7-1.9.7h-2.2v-7.6h2.3c.7 0 1.3.2 1.7.6.5.5.7 1 .7 1.7v2.7zm53.6-9.6h4.9v28.6h-4.9zm-184.6 4.7h4.7v23.9h4.9V14.5h4.7V9.8h-14.3zm61.8 23.9h12.8v-4.9h-7.9v-7.2h6.9v-4.7h-6.9v-7.1h7.9V9.8h-12.8zM78.4 32.5h4.3l1.1 5.9H89L82.7 9.8h-4.2l-6.4 28.6h5.2l1.1-5.9zm2.2-13.8l1.7 9.1H79l1.6-9.1zm296.3 13.8h4.3l1.1 5.9h5.2l-6.4-28.6h-4.2l-6.4 28.6h5.2l1.2-5.9zm2.1-13.8l1.7 9.1h-3.3l1.6-9.1zm61 14.8V9.8h-5v28.6h12.8v-4.9zm122.1-7l-2.7-16.6h-5.2l5.8 28.4h4.2l5.9-28.4h-5.2zm113.7 11.8h6.8c2 0 3.8-.8 5.2-2.3 1.2-1.4 1.8-3.1 1.8-5V16.7c0-2-.7-3.7-2.3-5.1-1.3-1.2-2.9-1.8-4.7-1.8h-6.7v28.5zm4.9-23.8h1.5c.7 0 1.2.2 1.6.6.5.4.7 1 .7 1.6V31c0 .9-.2 1.6-.6 2-.4.4-1 .6-1.9.6h-1.4V14.5zm-56.1-4.2c-2-1-4.4-.9-6.3.1-1 .5-1.9 1.3-2.5 2.2-.9 1.2-1.3 2.7-1.3 4.4v14.1c0 1.7.4 3.2 1.3 4.4.7.9 1.5 1.7 2.6 2.2 1 .5 2.1.8 3.3.8 1.1 0 2.1-.2 3.1-.7 1-.5 1.8-1.1 2.5-1.9 1-1.3 1.6-2.9 1.6-4.8V17c0-1.9-.5-3.5-1.6-4.8-.8-.8-1.7-1.4-2.7-1.9zm-.8 20.7c0 .9-.1 1.3-.3 1.5-.5.8-1.1 1.1-2 1.1-.7 0-1.3-.3-1.8-1-.3-.4-.4-.9-.4-1.7V17c0-.6.1-1 .3-1.3.5-.8 1.1-1.2 2-1.2.7 0 1.3.3 1.8.9.3.3.4.9.4 1.6v14zm127.3 7.3l-6.5-17 5.6-11.4h-5l-5.1 10.9V9.9h-4.9v28.4h4.9v-8.5l1.6-3 4.1 11.5zm48.2-5.8h4.3l1.1 5.9h5.2l-6.4-28.6h-4.2L793 38.4h5.2l1.1-5.9zm2.2-13.8l1.7 9.1h-3.3l1.6-9.1z"/></svg>
```

Comment:
How come we went back to this `.svg` approach? I still prefer the include.

Reply:
because it never works

Reply:
`never` is pretty inaccurate